test/browser_tests/page_objects/page_blog-single.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+function BlogSingle() {
+
+  this.get = function() {
+    browser.get( '/blog/lessons-weve-learned/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
```

Comment:
I know this is in all our tests, but just now wondering why this can't be `this.pageTitle = browser.getTitle();`? Anyway, if that's updated it should happen across the board in another PR.